### PR TITLE
OCPBUGS-44889: Allow external configuration of download host for production builds

### PR DESCRIPTION
**Pull request by joepvd**
Would need to go in tandem with https://github.com/openshift-eng/ocp-build-data/pull/5724

### CODE REVIEW

**Comment by joepvd** [comment]
/retitle OCPBUGS-44889: Allow external configuration of download host for production builds

**Comment by joepvd** [comment]
/hold
to become konflux ready, it is better to inject url and shasum at rebase time.

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 FROM registry.ci.openshift.org/ocp/4.18:installer AS builder
 
 ARG DIRECT_DOWNLOAD=false
+ENV ISO_HOST=https://example.com
```
Comment (dtantsur) [inline Dockerfile:4]: Cannot you default ISO_HOST to the same value for now to make the change mergeable?

**Pull request merged**